content/yaml-quick-start/building-a-unity-app.md

```diff
@@ -0,0 +1,492 @@
+---
+title: Building Unity mobile apps
+description: How to build Unity mobile apps with codemagic.yaml
+weight: 12
+
+---
+
+Unity is a cross-platform game engine developed by Unity Technologies. It can be used to create mobile applications that run on iOS and Android.
+
+Codemagic's Unity premium macOS base image has the Unity SDK, iOS and Android modules pre-installed. 
+
+This guide will show you how to configure a workflow that builds and publishes your app to TestFlight and Google Play using a **codemagic.yaml** configuration file. 
+
+{{<notebox>}}
+**Important!** Building Unity apps on Codemagic is currently in closed beta release. If you are interested in taking part in the beta, please contact us [here](https://codemagic.io/contact/)
+{{</notebox>}}
+
+## Adding your Unity project to Codemagic
+
+The apps you have available on Codemagic are listed on the Applications page. Click **Add application** to add a new app.
+
+1. On the Applications page, click **Set up build** next to the app you want to start building. 
+2. On the popup, select **Other** as the project type and click **Continue**.
+3. In your IDE, create a [`codemagic.yaml`](./yaml). This will be used to create our workflow.
+4. Commit the configuration file to the root of your repository.
+5. Back in app settings in Codemagic, scan for the `codemagic.yaml` file by selecting a **branch** to scan and clicking the **Check for configuration file** button at the top of the page. Note that you can have different configuration files in different branches.
+
+## Environment variables for Unity
+
+You will need to set the following environment variables for Unity specific values in Codemagic: 
+
+`UNITY_HOME`, `UNITY_SERIAL`, `UNITY_USERNAME` and `UNITY_PASSWORD`.
+
+You can add these as global environment variables for your team as follows (make sure the secure option is checked for any sensitive values):
```

Comment:
If it's possible to set up Unity only in teams, this should be listed as a requirement too